### DISCUSSION TITLE
ViewerContentToolsProvider - descriptions are missing

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#5978] [Q&A] [answered]
Hello everyone, good morning

I recently updated my application to version 3.7.2, in this update the installed version of @itwin/web-viewer-react is ^3.1.4

When I go to use ViewerContentToolsProvider it doesn't show me the descriptions of the features as per the attached image.

Furthermore, once I have used a Tool selection, how do I return to the original view?

Thanks for your precious help
Paul
<img width="142" alt="Mesure Tool" src="https://github.com/iTwin/itwinjs-core/assets/57446652/5404031b-7c11-4e0c-a8ea-e8ab774cea35">
<img width="132" alt="section tool" src="https://github.com/iTwin/itwinjs-core/assets/57446652/a04e7aa8-6f40-4882-a311-2a4b9f8c537b">

**Comment by aruniverse** [comment]
Are you using bentley/react-scripts? If so, what version are you on?
What version are you upgrading from?

**Comment by Paolo-lab** [comment]
hi yes I'm using @bentley/react-scripts": "^5.0.1

Thank you

**Reply by aruniverse** [reply]
are any other tools getting localized correctly? or is it just this one?

**Comment by Paolo-lab** [comment]
I'm leaving 3.0.2, so the opening of the view changed and I had to import
import {
   TreeWidget,
   TreeWidgetUiItemsProvider,
} from "@itwin/tree-widget-react";
import {
   PropertyGridManager,
   PropertyGridUiItemsProvider,
} from "@itwin/property-grid-react";

**Comment by Paolo-lab** [comment]
HI
@aruniverse thanks for the help.
It seems to me that the tools part also has description problems, do you attribute this to a localization problem?
<img width="318" alt="tool" src="https://github.com/iTwin/itwinjs-core/assets/57446652/2e14aead-cab1-4c2c-8920-a7157b1f7cbe">

**Reply by aruniverse** [reply, marked as answer]
Looks like assets arent being properly delivered in your app. Can you make sure you are using the latest version of bentley/react-scripts?
